models/like.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require ('mongoose');
+const { Schema } = mongoose;
+
+const LikeSchema = new Schema ({
+    type: String,
+    author: { type: Schema.Types.ObjectId, ref: 'User' },
+    postItem: { type: Schema.Types.ObjectId, ref: 'Post' }
```

Comment:
you can also specify the index like this
```suggestion
    postItem: { type: Schema.Types.ObjectId, ref: 'Post', index: true }
```

Reply:
Oh, that's cool! Just curious - how does this interact with having an unique index on all three fields?